travis/set_tags.sh

```diff
@@ -0,0 +1,13 @@
+if [ -z "$TRAVIS_TAG" ]; then
```

Comment:
Who would set this and in which case? Is this the tag we push to? (If so, where is it set). Is it the tag we "listen to" (like #master) to determine if we need to build a new "release"?